Tests/test_PDB_MMCIF2Dict.py

```diff
@@ -26,37 +26,305 @@
+                                                       "13.681", "26.728", "10.004",
+                                                       "30.553", "23.569", "10.927",
+                                                       "17.983", "8.191", "32.095",
+                                                       "11.520", "13.249", "15.919",
+                                                       "11.187", "16.743"])
+        # Turn black code style on
+        # fmt: on
         self.assertEqual(
             mmcif["_struct_ref.pdbx_seq_one_letter_code"],
-            [textwrap.dedent("""\
+            [
+                textwrap.dedent(
+                    """\
             GARASVLSGGELDKWEKIRLRPGGKKQYKLKHIVWASRELERFAVNPGLLETSEGCRQILGQLQPSLQTGSEELRSLYNT
             IAVLYCVHQRIDVKDTKEALDKIEEEQNKSKKKAQQAAADTGNNSQVSQNYPIVQNLQGQMVHQAISPRTLNAWVKVVEE
             KAFSPEVIPMFSALSEGATPQDLNTMLNTVGGHQAAMQMLKETINEEAAEWDRLHPVHAGPIAPGQMREPRGSDIAGTTS
             TLQEQIGWMTHNPPIPVGEIYKRWIILGLNKIVRMYSPTSILDIRQGPKEPFRDYVDRFYKTLRAEQASQEVKNWMTETL
             LVQNANPDCKTILKALGPGATLEEMMTACQGVGGPGHKARVLAEAMSQVTNPATIMIQKGNFRNQRKTVKCFNCGKEGHI
             AKNCRAPRKKGCWKCGKEGHQMKDCTERQANFLGKIWPSHKGRPGNFLQSRPEPTAPPEESFRFGEETTTPSQKQEPIDK
-            ELYPLASLRSLFGSDPSSQ""")])
+            ELYPLASLRSLFGSDPSSQ"""
+                )
+            ],
+        )
 
     def test_underscores(self):
         # Test values starting with an underscore are not treated as keys
         filename = "PDB/4Q9R_min.cif"
         mmcif = MMCIF2Dict(filename)
         self.assertEqual(len(mmcif.keys()), 5)
-        self.assertEqual(mmcif["_pdbx_audit_revision_item.item"], ["_atom_site.B_iso_or_equiv", "_atom_site.Cartn_x", "_atom_site.Cartn_y", "_atom_site.Cartn_z"])
+        self.assertEqual(
+            mmcif["_pdbx_audit_revision_item.item"],
+            [
+                "_atom_site.B_iso_or_equiv",
+                "_atom_site.Cartn_x",
+                "_atom_site.Cartn_y",
+                "_atom_site.Cartn_z",
+            ],
+        )
 
     def test_quotefix(self):
         # Test quote characters parse correctly
         filename = "PDB/1MOM_min.cif"
         mmcif = MMCIF2Dict(filename)
         self.assertEqual(len(mmcif.keys()), 21)
-        self.assertEqual(mmcif["_struct_conf.pdbx_PDB_helix_id"], ["A", "A'", "B", "C", "B'", "D", "E", "C'", "F", "G", "H", "D'", "E'", 'A\'"', "BC", "CD", "DE"])
+        self.assertEqual(
+            mmcif["_struct_conf.pdbx_PDB_helix_id"],
+            [
+                "A",
+                "A'",
+                "B",
+                "C",
+                "B'",
+                "D",
+                "E",
+                "C'",
+                "F",
+                "G",
+                "H",
+                "D'",
+                "E'",
+                "A'\"",
+                "BC",
+                "CD",
+                "DE",
+            ],
+        )
```

Comment:
This is an example where I don't like black's output - but it doesn't seem bad enough to disable black for this method.